packages/component-library/src/utils/DataChecker.js

```diff
@@ -0,0 +1,74 @@
+import React, { Fragment } from "react";
+import PropTypes from "prop-types";
+import { css } from "emotion";
+import checkData from "./checkData";
+
+const wrapperStyle = css`
+  margin: 0 auto;
+  max-width: 900px;
+  width: 90%;
+`;
+
+const DataChecker = ({ data, dataAccessors, message, children }) => {
+  const keys = Object.keys(dataAccessors);
+  const values = Object.values(dataAccessors);
+  const results = checkData(data, values);
+  const component = results.allKeysValid ? (
+    children
+  ) : (
+    <div className={wrapperStyle}>
+      <h1>{message}</h1>
+      <h2>Data Accessors</h2>
+      {keys.map((key, index) => {
+        const isValid =
+          results[values[index]].valid === results[values[index]].total;
+        const validCount = results[values[index]].valid;
+        const totalCount = results[values[index]].total;
+        const keyValue = values[index];
+        const invalidStyle = !isValid
+          ? css`
+              color: red;
+            `
+          : css``;
+        return (
+          <pre className={invalidStyle}>
```

Comment:
A `key` attribute should be added here to the `pre` element.

Reply:
Good catch